hooks/020-extra-files.chroot

```diff
@@ -35,3 +35,6 @@ mkdir -p /var/lib/cloud
 
 echo "ensure snapctl is available"
 ln -s ../lib/snapd/snapctl /usr/bin/snapctl
+
+echo "creating motd-news file"
+touch /var/cache/motd-news
```

Comment:
I believe we need to make /var/cache generally writable for the core snap, not special-case /var/cache/motd-news.

Reply:
Agree +1